polkadot-parachains/integritee-runtime/build.rs

```diff
@@ -16,14 +16,10 @@
 
 #[cfg(feature = "std")]
 fn main() {
-	substrate_wasm_builder::WasmBuilder::new()
-		.with_current_project()
-		.export_heap_base()
-		.import_memory()
+	substrate_wasm_builder::WasmBuilder::init_with_defaults()
+		.enable_metadata_hash("TEER", 12)
```

Comment:
@clangenb why not TEER-k?
I guess TEER-p will generally have different metadata
What are the implications of that choice?

Reply:
The implications are just that it is supposed to be the token symbol that you can retrieve with the rpc:

![image](https://github.com/integritee-network/parachain/assets/37865735/cb85f571-8cc7-4320-8086-aeaa7971fabe)